profile/README.md

```diff
@@ -0,0 +1,132 @@
+# Welcome, Apes!
+
+We are [ApeWorX LTD](https://apeworx.io), a collective of infrastructure engineers, smart contract devs, and Web3-curious folks,
+building the next generation of Python tooling and infrastructure to support your next great idea.
+Our suite of open source projects and developer tools, educational conten, and hosted services aims to guide you along your
```

Comment:
Are we no longer adhering the md style rule of 1 sentence per line? It's cool if we aren't but just wondering.